packages/hawtio/src/plugins/shared/connect-service.ts

```diff
@@ -62,35 +63,44 @@ export const SESSION_KEY_CURRENT_CONNECTION = 'connect.currentConnection'
 export const PARAM_KEY_CONNECTION = 'con'
 export const PARAM_KEY_REDIRECT = 'redirect'
 
-const LOGIN_PATH = '/connect/login'
+const PATH_LOGIN = '/connect/login'
+const PATH_PRESET_CONNECTIONS = 'preset-connections'
```

Comment:
Shouldn't it be also an URI with slash? (to make it less context-dependent)?

Reply:
Thanks, you are right. Fixed.